system-tests/_support/index.js

```diff
@@ -159,6 +159,30 @@ Cypress.addChildCommand("contents", function(elements) {
     .get();
 });
 
+/* Sets given JSONString as Value for ACE Editor
```

Comment:
We normally begin JDOC comment blocks with `/**` and on the next line you start document. Check the example above.